reprMathStatistics.ttl

```diff
@@ -1,83 +1,128 @@
+.
+<http://sweetontology.net/reprMathStatistics#ChiSquareTest>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#HypothesisTest> ;
+  rdfs:subClassOf [
+      rdf:type owl:Restriction ;
+      owl:allValuesFrom <http://sweetontology.net/reprMathFunction#ChiSquare> ;
+      owl:onProperty <http://sweetontology.net/relaMath#hasStatisticalDistribution> ;
+    ] ;
+.
+<http://sweetontology.net/reprMathStatistics#GeneralLinearModel>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#Regression> ;
+.
+<http://sweetontology.net/reprMathStatistics#GoodnessOfFit>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#Statistics> ;
+.
+<http://sweetontology.net/reprMathStatistics#HypothesisTest>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#StatisticalInference> ;
+.
+<http://sweetontology.net/reprMathStatistics#OLS>
+  rdf:type owl:Class ;
+.
+<http://sweetontology.net/reprMathStatistics#OrdinaryLeastSquares>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#Regression> ;
+  owl:equivalentClass <http://sweetontology.net/reprMathStatistics#OLS> ;
+.
+<http://sweetontology.net/reprMathStatistics#Regression>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#StatisticalOperation> ;
+.
+<http://sweetontology.net/reprMathStatistics#SampleStatistics>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#Statistics> ;
+.
+<http://sweetontology.net/reprMathStatistics#Sampling>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#StatisticalOperation> ;
+  rdfs:subClassOf <http://www.w3.org/ns/sosa/Sampling> ;
+.
+<http://sweetontology.net/reprMathStatistics#StatisticalDistribution>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathFunction#DistributionFunction> ;
+.
+<http://sweetontology.net/reprMathStatistics#StatisticalInference>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathStatistics#StatisticalOperation> ;
+.
+<http://sweetontology.net/reprMathStatistics#StatisticalOperation>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/reprMathOperation#Operation> ;
+.
+<http://sweetontology.net/reprMathStatistics#StatisticalSample>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Sample> ;
+  rdfs:subClassOf <http://www.w3.org/ns/sosa/Sample> ;
+  rdfs:subClassOf [
+      rdf:type owl:Restriction ;
+      owl:hasValue <http://sweetontology.net/stateSystem#Random> ;
+      owl:onProperty <http://sweetontology.net/rela#hasState> ;
+    ] ;
+  skos:note "Is the subclass relationship from StatisticalSample to humanResearchSample the right way round here? Or are they both subclasses of sosa:Sample? " ;
```

Comment:
We need to clarify this...

Reply:
Maybe 'Sample' (noun) is missing from SWEET

Reply:
SWEET may be a bit bound up wrt 'Sample'. Here a sample is a statistical process; elsewhere it would be a physical artifact (unless there is a different word used for that?). A humanResearch#Sample could be either one (computational or physical).  
  In any case, a statisticalSample can clearly be made from data other than humanResearch data, so it seems to me making it a subclass of humanResearch#Sample, with all the implications of that, is just incorrect. Other way around might be correct, or might not, depending on above.

Reply:
Yeah - SWEET is inconsistent here - 
humanResearch#Sample is the verb (act of sampling) 
whereas 
reprMathStatistics#StatisticalSample is the noun (output of act of sampling)

Reply:
I don't think what you said is a conflict in and of itself, but it reflects the general lack of a strategy for disambiguation. But this case will be wrong either way. I can't actually think of any way to make it right, I think this term (humanResearch#Sample) should be deprecated.

But geez, while we're at it, an Observation (next comment) is not a subclass of an Investigation, is it? I'd expect this to be a partOf relationship (whether by Observation we mean 'conclusion', 'single information point'). Unless we mean some kind of mini-investigation, but that is a tertiary meaning of Observation in my world.

Reply:
SWEET has generally used subsumption for everything ...